### PR TITLE
Event timeline colors

### PR DESCRIPTION
**Pull request by carterv**
Still only using event id, but the UI is complete and adding event source will be relatively easy. 

I also refactored the event timeline section a bit. Functionally it's still the same, I just felt like it could be organized better. 

Relates to #41

### CODE REVIEW

File: winlogtimeline/util/project.py [hunk 1]
```diff
@@ -1,34 +1,44 @@
+import json
 import os
 import sqlite3 as sql
 from datetime import datetime
 
+import util
```
Comment (ShaneKent) [inline winlogtimeline/util/project.py]: This line:
`import util`
is throwing me an error.

Comment (carterv) [reply winlogtimeline/util/project.py]: Relative imports are funky. Fixed it along with a few other things in this next change.

**Comment by ShaneKent** [comment]
It appears as though I can't access the timeline color changing options. Am I doing something wrong?
screen shot 2018-01-13 at 4 06 22 pm

**Comment by carterv** [comment]
@ShaneKent I can't seem to replicate that. The tools menu should be enabled as soon as a project is opened or created.

**Comment by ShaneKent** [comment]
Take a look at this page [here](https://stackoverflow.com/questions/24207870/cant-reenable-menus-in-python-tkinter-on-mac). It seems as though this has been a known issue with Macs for a while. 

Perhaps this would be a good time to do a "OS-type check." If the running OS is Windows, then we allow the menu bar to be disabled. If the OS is macOS, then we don't disable the menu but instead provide a little popup that says to open or create a project before being able to change the coloring.

**Comment by carterv** [comment]
@ShaneKent I'm fine with that, but I feel like that change is out of scope for this pull request. I'll create a separate issue for that.

**Pull request merged**